src/development/add-to-app/ios/add-flutter-screen.md

```diff
@@ -107,13 +143,56 @@ class AppDelegate: FlutterAppDelegate { // More on the FlutterAppDelegate.
 
 ### Show a FlutterViewController with your FlutterEngine
 
+{% samplecode vc %}
+
+{% sample SwiftUI %}
+The following example shows a generic `ContentView` with a
+`Button` hooked to present a [`FlutterViewController`][].
+The `FlutterViewController` uses the `FlutterEngine` instance
+passed in as an `EnvironmentObject`.
```

Comment:
Nit: This sounds like `FlutterViewController` requires an environment object. 
Also it's the FlutterDependencies, not flutter engine that gets injected. 

Maybe split into 2 sentences to make it more clear: 
1. the flutter view controller's constructor takes the pre-warmed engine as an argument 
2. this particular example uses environment object to inject flutter dependencies, which contains the flutter engine (this sentence is not as important, since you can also use singleton instead of dependency injection).

Reply:
Good call, thanks!